app/helpers/application_helper/toolbar/flavor_center.rb

```diff
@@ -1,4 +1,22 @@
 class ApplicationHelper::Toolbar::FlavorCenter < ApplicationHelper::Toolbar::Basic
+  button_group('flavor_configuration', [
+    select(
+      :flavor_configuration_choice,
+      'fa fa-cog fa-lg',
+      t = N_('Configuration'),
+      t,
+      :items => [
+        button(
+          :flavor_delete,
+          'pficon pficon-delete fa-lg',
+          t = N_('Remove Flavor'),
```

Comment:
@Loicavenel should this be `Remove Flavor` or rather `Remove Flavor from Inventory` ?

Reply:
Actually, never mind. It seems the flavor delete would also delete the flavor on the provider. So the above string is OK.